eq-author/src/App/QuestionnairesPage/QuestionnairesTable/Row.js

```diff
@@ -14,65 +15,81 @@ import QuestionnaireLink from "App/QuestionnairesPage/QuestionnaireLink";
 import FormattedDate from "App/QuestionnairesPage/FormattedDate";
 import FadeTransition from "components/transitions/FadeTransition";
 
+import { partial } from "lodash";
+import { rgba } from "polished";
+import { buildQuestionnairePath } from "utils/UrlUtils";
+
 const TruncatedQuestionnaireLink = Truncated.withComponent(QuestionnaireLink);
 TruncatedQuestionnaireLink.displayName = "TruncatedQuestionnaireLink";
 
+export const ShortTitle = styled.span`
+  color: ${colors.textLight};
+  text-decoration-color: ${colors.textLight};
+  font-size: 0.8em;
+  font-weight: bold;
+  text-transform: uppercase;
+  letter-spacing: 0.05em;
+`;
+
 const ButtonGroup = styled.div`
   display: flex;
-  justify-content: space-evenly;
 `;
 
-const TR = styled.tr`
-  border-bottom: 1px solid #e2e2e2;
+export const TR = styled.tr`
   border-top: 1px solid #e2e2e2;
+  border-bottom: 1px solid #e2e2e2;
+  background-color: ${props =>
```

Comment:
Can we do this with an `nth-of-type` selector instead of passing it down as a prop. As its purely used for display CSS seems a more appropriate location for it.
https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-of-type

Reply:
Done.